src/modules/ROOT/pages/changelog/202309/03.json

```diff
@@ -0,0 +1,255 @@
+{
+  "meta": {
+    "releasedOn": "2024.02.13"
+  },
+
+
+  "added": [
+    {
+      "what": [
+        "the ability to export a record field."
+      ],
+      "why": [
+        "to allow users to easily export a record field."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/3ed3b1ebb828b78f4ac834acdf7e6a100729dc80"
+      ]
+    },
+    {
+      "what": [
+        "light and dark themes as well as the ability to set custom colors for theming variables.",
+        "Added the option to set a branding color when configuring a page block, record field or reporter metric."
+      ],
+      "why": [
+        "to allow provide the user with more freedom to customize {PRODUCT_NAME}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1579",
+        "https://github.com/cortezaproject/corteza/pull/1671"
+      ]
+    },
+    {
+      "what": [
+        "missing `Manage schema alterations` translations in {APP_NAME_ADMIN}'s System component permissions."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/82f21b7007353c9dfcae4d61bc218f37aff28960"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the color of the alert for deleting a record from turquaz to yellow."
+      ],
+      "why": [
+        "to better fit the {PRODUCT_NAME} UI."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1634"
+      ]
+    },
+    {
+      "what": [
+        "the default time it take to load applications in {APP_NAME_ONE}."
+      ],
+      "why": [
+        "to make sure applications have been fetched and shown."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/8c2eec7f7c111c5319dfa89688561ac9966e80db"
+      ]
+    },
+    {
+      "what": [
+        "delimiter class for multi value fields to show values into new lines instead of spaces."
+      ],
+      "why": [
+        "to be consistent with the rest of {PRODUCT_NAME} UI."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/d82c5e6400b470e89a31c1668c7b9347c19d34dd"
+      ]
+    },
+    {
+      "what": [
+        "how sorting looks across webapps to match the UI in {APP_NAME_COMPOSE}."
+      ],
+      "why": [
+        "to standardize how sorting looks in all webapps."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1634"
+      ]
+    },
+    {
+      "what": [
+        "the way:",
+        "- permissions are evaluated",
+        "- workflows are executed",
+        "- expressions evaluation and objects are assignment",
+        "- state is managed and manipulated",
+        "- stack trace management and storage are executed",
+        "- IDs are generated in workflows"
+      ],
```

Comment:
This doesn't explain any of the changes so it's not a good message.
Since all this is mostly internal I'd suggest omitting all of this and using some generic improved performance like message